tests/components/sabnzbd/const.py

```diff
@@ -0,0 +1,41 @@
+"""Constants for the SABnzbd tests."""
+
+QUEUE_DATA = {
```

Comment:
This feels like it could be a fixture instead

Reply:
Sure 🙂